### ISSUE TITLE
Redo MHCflurry preprint with current code base

### ISSUE THREAD

**Issue by hammer** [hammerlab/mhcflurry#40]
In Tim's [validation notebook](https://github.com/hammerlab/mhcflurry-icml-compbio-2016/blob/master/notebooks/validation.ipynb), he uses [`combined_test_BLIND_dataset_from_kim2013.csv`](https://github.com/hammerlab/mhcflurry-icml-compbio-2016/blob/master/data/combined_test_BLIND_dataset_from_kim2013.csv) as his validation set.

We should have some code and documentation that describes what's in this file and how to build it from publicly available data.

**Comment by timodonnell** [comment]
Generalizing this issue a bit into redoing the model selection and evaluation in [Predicting Peptide-MHC Binding Affinities With Imputed Training Data](http://biorxiv.org/content/early/2016/05/22/054775) using the current MHCflurry code base. We should also publish these numbers with each models release.

**Comment by timodonnell** [comment]
This will be a new paper instead
